src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,58 +1,377 @@
+ * subsystems, commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
 
-  // Replace with CommandPS4Controller or CommandJoystick if needed
-  private final CommandXboxController m_driverController =
-      new CommandXboxController(OperatorConstants.kDriverControllerPort);
+    /* Controllers */
+    private final Joystick driver = new Joystick(0);
+    public static boolean centric = false;
+
+   /* Driver Controls */
+	private final int translationAxis = XboxController.Axis.kLeftY.value;
+	private final int strafeAxis = XboxController.Axis.kLeftX.value;
+	private final int rotationAxis = XboxController.Axis.kRightX.value;
+
+  // private final int translationAxis = XboxController.Axis.kRightX.value;
+	// private final int strafeAxis = XboxController.Axis.kLeftX.value;
+	// private final int rotationAxis = XboxController.Axis.kLeftY.value;
+
+    /* Driver Buttons */
+    private final JoystickButton zeroGyro = new JoystickButton(driver, XboxController.Button.kY.value);
+    private final JoystickButton robotCentric = new JoystickButton(driver, XboxController.Button.kLeftBumper.value);
+
+    private final JoystickButton dampen = new JoystickButton(driver, XboxController.Button.kRightBumper.value);
+
+    private final POVButton up = new POVButton(driver, 90);
+    private final POVButton down = new POVButton(driver, 270);
+    private final POVButton right = new POVButton(driver, 180);
+    private final POVButton left = new POVButton(driver, 0);
+
+    /* Subsystems */
+    private final Swerve s_Swerve = new Swerve();
+    private final PoseEstimator s_PoseEstimator = new PoseEstimator();
+    private static Shooter shooter;
+
+
+  // Subsystems
+
+  // Controller
+  private static final CommandXboxController driverController =
+      new CommandXboxController(Constants.kDriverControllerPort);
+  private static final CommandXboxController operatorController =
+      new CommandXboxController(Constants.kOperatorControllerPort);
+
+  private static Intake intake;
+private static Arm arm = new Arm();
+
+
+  // Dashboard inputs
+
+  
+
+    /** The container for the robot. Contains subsystems, OI devices, and commands. */
+    public RobotContainer() {
+        s_Swerve.setDefaultCommand(
+            new TeleopSwerve(
+                s_Swerve, 
+                () -> -driver.getRawAxis(translationAxis), 
+                () -> -driver.getRawAxis(strafeAxis), 
+                () -> -driver.getRawAxis(rotationAxis), 
+                () -> centric,
+                () -> dampen.getAsBoolean(),
+                () -> 1 //speed multiplier 
+            )
+        );
 
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
-  public RobotContainer() {
-    // Configure the trigger bindings
-    configureBindings();
+        intake = new Intake();
+        shooter = new Shooter();
```

Comment:
Be consistent with how you initialize systems.  intake and shooter are assigned here in the constructor, but you did a static init of the arm system.  Keep the inits together.